cmd/tailscale/main.go

```diff
@@ -591,6 +591,18 @@ func (a *App) getBugReportID(ctx context.Context, bugReportChan chan<- string, f
 	bugReportChan <- string(logBytes)
 }
 
+func (a *App) login(ctx context.Context, errHandler func()) {
+	ctx, cancel := context.WithDeadline(ctx, time.Now().Add(2*time.Second))
+	defer cancel()
+	r, err := a.localAPIClient.Call(ctx, "POST", "login-interactive", nil)
+	defer r.Body().Close()
+
+	if err != nil {
+		log.Printf("login: %s", err)
+		errHandler()
```

Comment:
Instead of having to pass in an error handler here (which is a bit unidiomatic in Go), you could just store the backend on the app itself and then call `a.b.StartLoginInteractive()` here.

Reply:
Sounds good